changedetectionio/tests/test_xpath_selector.py

```diff
@@ -277,3 +277,45 @@ def test_check_with_prefix_include_filters(client, live_server):
     assert b"Some text that will change" not in res.data #not in selector
 
     client.get(url_for("form_delete", uuid="all"), follow_redirects=True)
+
+def test_xpath_20(client, live_server):
```

Comment:
BTW, Isn't your intention the xpath2.0-3.1 syntax test?

Reply:
New commit(https://github.com/dgtlmoon/changedetection.io/pull/1774/commits/37bcc893423ba2bc3f7998fe7efc9144361384e0) tests XPath2.0-3.1 functions.